partiql-playground/README.md

```diff
@@ -0,0 +1,42 @@
+# PartiQL Playground (Proof of Concept)
+
+PartiQL Playground is intended to provide the required code for enabling execution of PartiQL queries on web.
+
+_Please note, at this stage the code within this package is considered experimental and should be used for production._
+
+## Local Usage
+For local usage follow the below steps.
+
+1. Pull down the `partiql-rust` package from GitHub:
+```bash
+git clone --recursive https://github.com/partiql/partiql-lang-rust.git
+```
+2. Enter the `partiql-playground` root directory:
+```bash
+cd partiql-lang-rust/partiql-playground
+```
+3. Start a webserver from the root directory, as an example, you can use [Python's SimpleHTTPServer](https://docs.python.org/3.8/library/http.server.html#http.server.SimpleHTTPRequestHandler):
```

Comment:
Btw, I had to run the `wasm-pack build --target web` step from the "Development" section below to get the query editor to show up. Might be worth including as a step before starting the server.

Reply:
Ah, that's a miss from my side. Added.